tests/stat/test_stat.c

```diff
@@ -29,6 +30,11 @@ void create_file(const char *path, const char *buffer, int mode) {
   close(fd);
 }
 
+void active_sleep(int delta) {
+    time_t end = time(NULL) + delta;
+    while (time(NULL) < end);
+}
```

Comment:
If possible its best if we can avoid buys waiting in tests.

Can you perhaps explicitly set the mtime of a directory to be in the past to ensure it gets correctly updated, rather than sleeping/waiting?

Reply:
sure, did it using the same technic as the other tests in `test_stat.c`